projects/vaults/contracts/vaults/Symbiotic/InceptionVault_S.sol

```diff
@@ -265,16 +309,25 @@ contract InceptionVault_S is MellowHandler, IInceptionVault_S {
     function flashWithdraw(
         uint256 iShares,
         address receiver
-    ) external whenNotPaused nonReentrant {
+    ) public whenNotPaused nonReentrant {
```

Comment:
why not external?

Reply:
It is being used by ERC4626 withdraw. Or do you want to have an external call from EIP4626 Withdraw to flashWithdraw ?

Reply:
Oh wait. I see. Let me change to external since we have private _flashWithdraw